core-text/src/font_descriptor.rs

```diff
@@ -194,6 +194,10 @@ declare_TCFType! {
 impl_TCFType!(CTFontDescriptor, CTFontDescriptorRef, CTFontDescriptorGetTypeID);
 impl_CFTypeDescription!(CTFontDescriptor);
 
+// "Font objects (CTFont, CTFontDescriptor, and associated objects) can be used
+//  simultaneously by multiple operations, work queues, or threads."
```

Comment:
Can you link to the docs?

Reply:
Oh I missed this comment before I r+ it...